### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_MigrationAssessment_Keyboard:There is no visible indication of focus on the OK button.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/azuredatastudio#15671]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.7;#AINotInScope;#Keyboard;#Focusvisible

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and tab till view/select button and press enter.
7. Tab till Ok button.

### Actual:
There is no visible indication of focus on the OK button.
**Similar issue is observed throughout the app where OK button is present.**

### Expected:
The OK button should have a proper visible indication of focus.

### User Impact:
Keyboard only users will not be able to see where the focus is situated at.

**Comment by tripathiakhil429** [comment]
Closing as duplicate.

**Comment by 12-shweta** [comment]
**Github tags:**#Closed;
